Library/Formula/kibana.rb

```diff
@@ -1,48 +1,32 @@
 class Kibana < Formula
-  desc "Visualization tool for elasticsearch"
+  desc "Analytics and search dashboard for Elasticsearch"
   homepage "https://www.elastic.co/products/kibana"
-  url "https://github.com/elastic/kibana/archive/v4.1.1.tar.gz"
-  sha256 "3f91e99e20e82d4e84ec141007822fea8f9454c71595551f9348ea2609c98284"
+  url "https://download.elastic.co/kibana/kibana/kibana-4.3.0-darwin-x64.tar.gz"
```

Comment:
@DomT4 Thanks for the feedback here. I'll probably be looking at modifying this pull request to use a source release rather than a binary distribution but I might not get to it until next week.